xarray/core/duck_array_ops.py

```diff
@@ -278,6 +276,25 @@ def f(values, axis=None, skipna=None, **kwargs):
 cumsum_1d.numeric_only = True
 
 
+_mean = _create_nan_agg_method('mean')
+
+
+def mean(array, axis=None, skipna=None, **kwargs):
```

Comment:
I would like to make this compatible this function with CFTime index. @spencerkclark, could you comment for this?

Reply:
Thanks, @spencerkclark 

It would be nice if you could send a follow-up PR :)

Reply:
Sure thing, I'd be happy to take care of making this compatible with cftime dates.